packages/react-charting/src/types/IDataPoint.ts

```diff
@@ -311,7 +311,7 @@ export interface ILineChartPoints {
   /**
    * color for the legend in the chart
    */
-  color: string;
+  color?: string;
```

Comment:
Add a comment here that we fallback on the default color palette if color is not provided

Reply:
done